book/A-git-in-other-environments/sections/sublimetext.asc

```diff
@@ -0,0 +1,12 @@
+=== Git in Sublime Text
+
+From version 3.2 onwards, Sublime Text has git integration in the editor.
+
+The features are:
+*  The sidebar will show the git status of files and folders with a badge/icon.
+*  Files and folders that are in your .gitignore file will be faded out in the sidebar.
+*  In the status bar, you can see the current git branch and how many modifications you have made.
+*  All changes to a file are now visible via markers in the gutter.
+*  You can use part of the Sublime Merge git client functionality from within Sublime Text. (This requires that Sublime Merge is installed.)
+
+The official documentation can be found here: https://www.sublimetext.com/docs/3/git_integration.html
```

Comment:
If you add `[]` to the end of this line it'll become a link.

Reply:
Fixed link.